components/family-studies/api/src/main/java/org/phenotips/studies/family/Family.java

```diff
@@ -38,7 +39,7 @@
  * @version $Id$
  * @since 1.4
  */
-public interface Family
+public interface Family extends PrimaryEntity
```

Comment:
I think to make it more clear what `Family` is about we should remove all duplicate (available in `PrimaryEntity`) methods from this interface since they are inherited (`getId()`, `getXDocument()`, `getDocumentReference()`, `toJSON()`)

Reply:
Right, fixed.